### ISSUE TITLE
scaling is applied before cross validation

### ISSUE THREAD

**Issue by weishen89** [gastonstat/plsdepot#4]
Scaling should applied separately on the training set and the test set (using the mean and sd from the training set).
The current code provides an over-optimistic result.
